### ISSUE TITLE
Pink Petals are assembled with a new ship

### ISSUE THREAD

**Issue by Toadbean** [ValkyrienSkies/Eureka#229]
### This issue occurs when only Valkyrien Skies and addons are installed and no other mods

- [X] I have tested this issue and it occurs when no other mods are installed

### Minecraft Version

1.19

### Mod Loader

Fabric

### Issue description

Whenever you assemble a new ship with a ship helm, pink petals will be assembled with the ship if they are touching any of it's blocks. 

### Issue reproduction

Place an oak log next to a pink petals block and place a ship helm on the oak block and assemble the ship.

**Comment by millennIumAMbiguity** [comment]
> ### Minecraft Version
> 1.19

[Pink Petals](https://minecraft.fandom.com/wiki/Pink_Petals) was added in Minecraft 1.20.

**Comment by Toadbean** [comment]
Yes of course, but there wasn't an option to put 1.20.1 when was writing the form, but I probably should have added it as a comment
